Detector/DetComponents/src/GeoConstruction.cpp

```diff
@@ -66,7 +66,7 @@ void GeoConstruction::ConstructSDandField() {
 G4VPhysicalVolume* GeoConstruction::Construct() {
   DD4hep::Simulation::Geant4Mapping& g4map = DD4hep::Simulation::Geant4Mapping::instance();
   DD4hep::Geometry::DetElement world = m_lcdd.world();
-  DD4hep::Simulation::Geant4Converter conv(m_lcdd, DD4hep::INFO);
+  DD4hep::Simulation::Geant4Converter conv(m_lcdd, DD4hep::DEBUG);
```

Comment:
wouldn't we rather pass Gaudi algorithm's output level? (as in [GeoSvc](https://github.com/HEP-FCC/FCCSW/pull/271/files#diff-6f8ea977a393a43c03b8aaf4e0eedd4eL28))

Reply:
Ah, no, I was confused by this as well. As far as I was I understand it the function of the line you linked is to say "print only messages with a level xyz or higher", while the argument here is saying "use xyz to print the message `Geant4Converter  INFO  ++ Volume     + CalEndcapCryoOuter_negative_SimpleCylinder converted: 0xd024d1e0 ---> G4: 0xf3967da0` in GeoConverter". So if we pass the Gaudi algorithm's output level this pretty pointless line (actually, thousands of them) will *always* be printed, because it always has level xyz (You can see it from the line I pasted - it has printLevel INFO!).

It seems fairly obvious to me that the Geant4Converter output should be DEBUG, so I don't know why the interface is like that...

Reply:
Ah, thanks for the clarification. Indeed it makes sense now!